src/main/java/com/dev/theater/confing/DataInitializer.java

```diff
@@ -0,0 +1,44 @@
+package com.dev.theater.confing;
+
+import com.dev.theater.model.Role;
+import com.dev.theater.model.User;
+import com.dev.theater.service.RoleService;
+import com.dev.theater.service.UserService;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    public DataInitializer(RoleService roleService, UserService userService) {
+        this.roleService = roleService;
+        this.userService = userService;
+    }
+
+    @PostConstruct
+    public void addAdmin() {
```

Comment:
let's make these 2 methods private and call them inside one `@PostConstruct` method
conceptually this is a bit confusing)